index.js

```diff
@@ -224,21 +225,40 @@ async function terraform(terraformDirectory, args) {
       exitCode = 1;
       break;
     }
+    core.setOutput('changes', tfd.status == 2 ? 'true' : 'false');
     core.endGroup();
     process.exit(exitCode);
   }
 
   core.startGroup('Run terraform plan');
-  const tfp = await terraform(terraformDirectory, ['plan', `-lock=${terraformLock}`, `-parallelism=${terraformParallelism}`, `-refresh=${terraformRefresh}`, '-out=terraform.plan'].concat(terraformTargets).concat(terraformVariableFiles).concat(terraformPlanDestroy));
+  const tfp = await terraform(terraformDirectory, ['plan', `-lock=${terraformLock}`, `-parallelism=${terraformParallelism}`, `-refresh=${terraformRefresh}`, '-out=terraform.plan', '-detailed-exitcode'].concat(terraformTargets).concat(terraformVariableFiles).concat(terraformPlanDestroy));
   core.endGroup();
-  if (tfp.status > 0) {
+  if (tfp.status == 1) { // see -detailed-exitcode documentation
```

Comment:
Perhaps just briefly state what the (expected) exit codes actually are instead (in case behavior would change and our expectations would no longer match the documentation).